dnf/plugin.py

```diff
@@ -193,7 +194,7 @@ def unload_removed_plugins(self, transaction):
 
         # check whether removed plugin file is added at the same time (upgrade of a plugin)
         for pkg in transaction.install_set:
-            erased_plugin_files.difference_update(pkg.files)
+            erased_plugin_files.difference_update(pkg._header[rpm.RPMTAG_FILENAMES])
```

Comment:
I believe that here we are with the rpm transaction completed, so the header should be always present, right?

Reply:
I think so as well, thanks. :slightly_smiling_face: